nautobot/dcim/tests/test_views.py

```diff
@@ -197,7 +201,7 @@ def setUpTestData(cls):
             "tenant": tenant.pk,
             "facility": "Facility X",
             "asn": 65001,
-            "time_zone": pytz.UTC,
+            "time_zone": zoneinfo.ZoneInfo("UTC"),
```

Comment:
[This migration guide that Django recommends](https://pytz-deprecation-shim.readthedocs.io/en/latest/migration.html#utc-and-fixed-offset-zones) says we should use `datetime.timezone.utc` as a replacement for `pytz.UTC` but I don't know what the difference is.

Reply:
Seems like it's safer to use the `datetime.timezone.utc` based on [this](https://stackoverflow.com/questions/74467999/why-does-zoneinfoutc-do-different-time-conversions-from-timezone-utc)

Reply:
I tried that initially but `datetime.timezone.utc` is a different object than `zoneinfo.ZoneInfo("UTC")`, and the object equality test failed with the former but passes with the latter.

Reply:
Wait, for a location this is probably totally irrelevant!